Source/SwiftLintFramework/Rules/FirebaseConfigActivateRule.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  FirebaseConfigActivateRule.swift
+//  SwiftLint
+//
+//  Created by Ibrahim Ulukaya (Google Inc.) on 3/8/17.
+//  Copyright © 2017 Realm. All rights reserved.
+//
+
+import SourceKittenFramework
+
+public struct FirebaseConfigActivateRule: ASTRule, RecursiveRule, OptInRule {
+
+    public var configuration = SeverityConfiguration(.warning)
+
+    public init() {}
+
+    public static let description = RuleDescription(
+        identifier: "firebase_config_activate",
+        name: "Firebase Config Activate",
+        description: "Firebase Config should be activated.",
+        nonTriggeringExamples: [
+            "remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+                " (status, error) -> Void in \n self.remoteConfig.activateFetched() \n }"
+        ],
+        triggeringExamples: [
+            "remoteConfig.fetch(withExpirationDuration: TimeInterval(expirationDuration)) {" +
+                " (status, error) -> Void in \n }"
+        ]
+    )
+
+    public func validate(file: File, kind: SwiftExpressionKind,
+                         dictionary: [String: SourceKitRepresentable]) -> [StyleViolation] {
+        guard SwiftExpressionKind.call == kind else {
+            return []
+        }
+
+        guard dictionary.name!.hasSuffix(".fetch") else {
```

Comment:
This will fail in case `name` is `nil`. Consider using guard to make sure `name` exists before using it.

```swift
guard let name = dictionary.name, name.hasSuffix(".fetch") else {
    // code...
}
```

Compare these examples:

```swift
extension Dictionary where Key: ExpressibleByStringLiteral {
    var name: String? {
        return self["name"] as? String
    }
}

func isValidDictionary1(_ dictionary: [String: String]) -> Bool {
    guard let name = dictionary.name, name.hasSuffix(".fetch") else { return false }
    return true
}

func isValidDictionary2(_ dictionary: [String: String]) -> Bool {
    guard dictionary.name!.hasSuffix(".fetch") else { return false }
    return true
}

isValidDictionary1(["name": "test.fetch"])   // true
isValidDictionary1(["aname": "test.fetch"])  // false

isValidDictionary2(["name": "test.fetch"])   // true
isValidDictionary2(["aname": "test.fetch"])  // fatal error: unexpectedly found nil while unwrapping an Optional value
```